src/components/DonarDashboard/DonarSuppotedProjects.tsx

```diff
@@ -127,7 +127,9 @@ const DonarSuppotedProjects = ({
                 {formatAmount(totalContributions) || 0} POL{' '}
               </span>
               <span className='font-medium text-[#82899A]'>
-                ~ $ {formatAmount(project.totalDonations) || 0}
+                ~ ${' '}
+                {formatAmount(Math.round(project.totalDonations * 100) / 100) ||
```

Comment:
I don't know what the purpose of this is; there is no description in PR or attached issue.
BTW, the ` number.toLocaleString` has these useful options
```ts
    interface NumberFormatOptions {
        roundingPriority?: "auto" | "morePrecision" | "lessPrecision" | undefined;
        roundingIncrement?: 1 | 2 | 5 | 10 | 20 | 25 | 50 | 100 | 200 | 250 | 500 | 1000 | 2000 | 2500 | 5000 | undefined;
        roundingMode?: "ceil" | "floor" | "expand" | "trunc" | "halfCeil" | "halfFloor" | "halfExpand" | "halfTrunc" | "halfEven" | undefined;
        trailingZeroDisplay?: "auto" | "stripIfInteger" | undefined;
    }

```